src/services/HelpService.js

```diff
@@ -0,0 +1,39 @@
+import Service from '../abc/Service';
+import {HELP_CONTENT} from "../actions";
+
+export default class HelpService extends Service {
+	constructor(app, serviceName="HeaderService"){
+		super(app, serviceName);
+		this.App = app;
+	}
+
+	async setData(path) {
+		let withExtension;
```

Comment:
@Aringoaway Choose better naming, from `whitExtension` one can have no clue, what does it do (if they dont read the whole code)